### DISCUSSION TITLE
[feature request]  Price [device stored] monitoring feature

### DISCUSSION THREAD

**Discussion by serpico** [openfoodfacts/openfoodfacts-androidapp#3972] [Ideas]
I've looked in the issues, but haven't found anything, anybody ever asked for this? It might be considered outside the scope of the app as well.

I've seen it on some feature's wishlist for OFF, not on github, but couldn't find it anymore.

Maybe a separate tab, which the user could enable through the Preferences, similar to the current contribution tab ( although, some text box should be easily accessible
for quickly adding the price ), maybe button(s) ( the price changed | the price is identical )  as a way to quickly add a "sample" without typing any number if the price is the same.

I could see a few neat potential uses ( calculate cost per serving based on the list feature for a recipe [could also ease meal swapping], add another level to the compare products function ( price difference per serving size, 100g, whatever or % of price between two similar products, one being deemed less healthy than the other ( according to NOVA, nutriscore,...), total shopping/grocery cart value over time ( real sample inflation).

I don't know if it changes much on the technical side, especially on server load if a sizeable portion of the user base start to look for the cheapest table salt in the universe, but local storage seems good enough for me at first glance.

**Comment by teolemon** [comment]
@serpico This could be interesting but requires resources we currently don't have. Documenting the plan on the wiki (https://wiki.openfoodfacts.org) or on a document, along with mockups, use cases… would be very interesting though.
